lib/hexo/post.js

```diff
@@ -247,7 +247,17 @@ class Post {
     }
 
     // disable Nunjucks when the renderer specify that.
-    const disableNunjucks = ext && ctx.render.renderer.get(ext) && !!ctx.render.renderer.get(ext).disableNunjucks;
+    let disableNunjucks = false;
+    let extRenderer = ext && ctx.render.renderer.get(ext);
+    if (extRenderer) {
+      disableNunjucks = extRenderer.disableNunjucks === true;
```

Comment:
```suggestion
      disableNunjucks =Boolean(extRenderer.disableNunjucks);
```

Will it work?

Reply:
works in sync [coffeescript](https://github.com/hexojs/hexo-renderer-coffeescript/pull/72/checks?check_run_id=1039930141), async [less](https://github.com/hexojs/hexo-renderer-less/pull/53/checks?check_run_id=1039936433) and [stylus](https://github.com/hexojs/hexo-renderer-stylus/pull/56/checks?check_run_id=1039941940).